src-backbone/app/css/modules/procedures.less

```diff
@@ -78,12 +78,16 @@ header#procedures-list-heading{
         margin: 30px 0;
     }
 
+    a#import-procedure-btn{
+        margin: 30px 0 0 10px;
+    }
+
     h2{
         line-height: 40px;
         float: left;
     }
 
-    a#new-procedure-btn{
+    a#new-procedure-btn, a#import-procedure-btn{
```

Comment:
Okay sure, didn't realize it was done like that else where. Low risk of another button in the same category being added elsewhere anyways.